diesel_cli/Cargo.toml

```diff
@@ -21,6 +21,7 @@ diesel = { version = "0.16.0", default-features = false }
 dotenv = ">=0.8, <0.11"
 diesel_infer_schema = "0.16.0"
 clippy = { optional = true, version = "=0.0.162" }
+libsqlite3-sys = { version = ">=0.8.0, <0.9.0", default-features = false, optional = true, features = ["bundled"] }
```

Comment:
This includes the `bundled` feature by default. I'm fine with giving a flag for this, but I don't think it should be the default.